argus/backend/plugins/sct/controller.py

```diff
@@ -126,6 +131,37 @@ def sct_events_submit(run_id: str):
     }
 
 
+@bp.route("/<string:run_id>/events/get", methods=["GET"])
+@api_login_required
+def sct_events_get(run_id: str):
+    limit = int(request.args.get("limit", 100))
+    before = request.args.get("before")
+    severities = request.args.getlist("severity")
+    result = SCTService.get_events(run_id=run_id, limit=limit, before=before, severities=severities)
```

Comment:
this logic is a bit off - see that limit 100 will be for each severity - some might have reached it and `before` filter must apply to the one that reached that limit. In result we'll fetch again the same events for these severities limit was not reached.

I think we need to separate severities `"/<string:run_id>/events/<string:severity>/get"` and later in ui, each level should be in own 'tab'. Otherwise 'before' filter won't have sense. WDYT?